samples/login.py

```diff
@@ -80,7 +86,7 @@ def sample_connect_to_server(args):
     server = TSC.Server(args.server, use_server_version=True, http_options={"verify": check_ssl_certificate})
     server.auth.sign_in(tableau_auth)
     server.version = "2.6"
-    new_site: TSC.SiteItem = TSC.SiteItem("cdnear", content_url=env.site)
```

Comment:
is there a better hardcoded name we can use?

Reply:
@jacalata good catch! Looking at it more closely what if we just remove this switchsite chunk to make the demo more clear?

```py
    server.version = "2.6"
    new_site: TSC.SiteItem = TSC.SiteItem("cdnear", content_url=args.site)
    server.auth.switch_site(new_site)
```

Reply:
yea we probably don't need that in an example called login

Reply:
Done!